docs/source/spelling_wordlist.txt

```diff
@@ -241,6 +248,8 @@ util
 utils
 uwsgi
 uWSGI
+validator
```

Comment:
Same here. I spelled it outloud multiple times to make sure that I wasn't going crazy.